python/lsst/ts/tunablelaser/hardware.py

```diff
@@ -470,11 +439,11 @@ class MaxiOPG:
         Handles the "Configuration" register.
     """
 
-    def __init__(self, commander, simulation_mode=False):
+    def __init__(self, commander, simulation_mode=False, configuration="No SCU"):
```

Comment:
I think I was mistaken. I see at least one place it is used. And with spaces in the value an enum would be a bit ugly (e.g.:
```
class LaserConfig(enum.Enum):
   """<description>"""
    NO_SCU = "No SCU"
```
but still well worth doing, I think. It catches typos early and also provides a useful place to document the options (in the doc string for the config class) for any that are not obvious. Too bad it requires using item.value when you pass choices to the laser.

For items such as "ON", "OFF" that can also be variable names you can skip the need to use item.value as follows:
```
class RegisterBool(str, enum.Enum):
    ON = "ON"
    OFF = "OFF"
```
You can pass RegisterBool.ON directly to the laser and it will see it as "ON".